src/applications/rx/components/Breadcrumbs.jsx

```diff
@@ -25,50 +26,28 @@ class Breadcrumbs extends React.Component {
     const crumbs = [
       <a href="/" key="home" onClick={() => { recordEvent({ event: 'nav-breadcrumb' });}}>Home</a>,
       <a href="/health-care/" key="healthcare">Health Care</a>,
+      <Link to="/" key="prescriptions">Prescription Refills</Link>
     ];
 
-    if (pathname.match(/\/\d+$/)) {
-      crumbs.push(<Link to="/" key="prescriptions">Prescription Refills</Link>);
+    if (pathname.match(/\/\d+\/?(track)?$/)) {
```

Comment:
@1Copenut could you please comment on these differences from the earlier version?